packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -128,7 +128,7 @@ export function getDomTreeShapes(element, rootNode) {
 
   while (element && element !== rootNode) {
     // We reach a Swipeable View, no need to look higher in the dom tree.
-    if (element.hasAttribute('data-swipeable')) {
+    if (typeof element.hasAttribute !== 'function' || element.hasAttribute('data-swipeable')) {
```

Comment:
This will cause it to never find the attribute, unless it's only particular elements that fail to have the `hasAttribute` function. Can you provide steps to reproduce the error you're receiving?

Reply:
I have found https://github.com/emberjs/ember.js/issues/16730 related to this problem but it's IE 11 related